src/ralph/settings/base.py

```diff
@@ -267,3 +267,5 @@ def os_env_true(var, default=''):
 MY_EQUIPMENT_LINKS = json.loads(os.environ.get('MY_EQUIPMENT_LINKS', '[]'))
 MY_EQUIPMENT_REPORT_FAILURE_URL = os.environ.get('MY_EQUIPMENT_REPORT_FAILURE_URL', '')  # noqa
 MY_EQUIPMENT_SHOW_BUYOUT_DATE = os_env_true('MY_EQUIPMENT_SHOW_BUYOUT_DATE')
+
+MAP_IMPORTED_ID_TO_NEW_ID = False
```

Comment:
Really easy to understand for new developer who came across this line. Really supportive.

http://38.media.tumblr.com/2dce58f90efefd95b2393275ea67116d/tumblr_inline_nbk7kdW9GK1raprkq.gif